spec/src/language_negotiation_lambda_spec.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+var languageNegotiationLambda = require( '../../src/language_negotiation_lambda' );
+
+describe( 'languageNegotiationLambda', function() {
+    var event = {
+        "Records": [
+            {
+              "cf": {
+                "request": {
+                  "headers": {
+                    "Host": [
+                      "localhost"
+                    ],
+                    "Cookie": [
+                      "SomeCookie=1; AnotherOne=A"
+                    ],
+                    "User-Agent": [
+                      "Test Agent"
+                    ]
+                  },
+                  "clientIp": "2001:cdba::3257:9652",
+                  "httpVersion": "2.0",
+                  "uri": "/",
+                  "method": "GET"
+                },
+                "config": {
+                  "distributionId": "EXAMPLE"
+                }
+              }
+            }
+        ]
+    };
+    var default_header = ['en'];
+
+    beforeEach(function(){
+        delete event.Records[0].cf.request.headers["Accept-Language"];
+        delete event.Records[0].cf.request.headers["X-Accept-Language"];
```

Comment:
Why not just create the event in the beforeEach? That way you're guaranteed that every test will start with a fresh version of it.